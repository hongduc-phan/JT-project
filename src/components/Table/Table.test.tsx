import React from 'react';
import renderer from 'react-test-renderer';

import {Table, TableHead, TableBody, TableRow, TableCell} from './index';

describe('<Sidebar />', () => {
  const TableConatiner = () => (
    <Table>
      <TableHead>
        <TableRow>
          <TableCell>Department</TableCell>
          <TableCell>Salary</TableCell>
        </TableRow>
      </TableHead>
      <TableBody>
        <TableRow>
          <TableCell>HR</TableCell>
          <TableCell>8000</TableCell>
        </TableRow>
        <TableRow>
          <TableCell>Sales</TableCell>
          <TableCell>4000</TableCell>
        </TableRow>
        <TableRow>
          <TableCell>Engineering</TableCell>
          <TableCell>7000</TableCell>
        </TableRow>
        <TableRow>
          <TableCell>Finance</TableCell>
          <TableCell>3000</TableCell>
        </TableRow>
      </TableBody>
    </Table>
  );

  it('Should match table snapshot', () => {
    const table = TableConatiner();
    const tree = renderer.create(table).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
