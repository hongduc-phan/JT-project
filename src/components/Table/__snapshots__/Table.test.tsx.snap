// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sidebar /> Should match table snapshot 1`] = `
<table
  className="table"
>
  <thead
    className="tableHead"
  >
    <tr>
      <th
        className="column"
      >
        <span
          className="typo body2 greyDark left"
        >
          Department
        </span>
      </th>
      <th
        className="column"
      >
        <span
          className="typo body2 greyDark left"
        >
          Salary
        </span>
      </th>
    </tr>
  </thead>
  <tbody
    className="tableBody"
  >
    <tr>
      <td
        className="cell"
      >
        HR
      </td>
      <td
        className="cell"
      >
        8000
      </td>
    </tr>
    <tr>
      <td
        className="cell"
      >
        Sales
      </td>
      <td
        className="cell"
      >
        4000
      </td>
    </tr>
    <tr>
      <td
        className="cell"
      >
        Engineering
      </td>
      <td
        className="cell"
      >
        7000
      </td>
    </tr>
    <tr>
      <td
        className="cell"
      >
        Finance
      </td>
      <td
        className="cell"
      >
        3000
      </td>
    </tr>
  </tbody>
</table>
`;
